docs/cli.md

```diff
@@ -194,73 +194,6 @@ analysis_models:
 the `model-analyzer profile` command when you do not want to include the results
 from a previous profile.
 
-## Subcommand: `analyze`
-
-**Note: This subcommand has been deprecated and is slated for removal. This
-subcommand's functionality has been subsumed into the profile subcommand**
-
-The `analyze` subcommand allows the user to create summaries and data tables
-from the measurements taken using the `profile` subcommand. The YAML config file
-can be used to set constraints and objectives used to sort and filter the
-measurements, and order the model configs and models according to the metrics
-collected. Use the
-following command to see the usage and argument descriptions for the subcommand.
-
-```
-$ model-analyzer analyze -h
-```
-
-The `analyze` subcommand begins by loading the "latest" checkpoint available in
-the checkpoint directory. Next, it sorts the models specified in the CLI or
-config YAML, provided they contain measurements in the checkpoint, using the
-objectives specified in the config YAML. Finally, it constructs summary PDFs
-using the top model configs for each model, as well as across models, if
-requested (See the [Reports](./report.md) section for more details). The
-`analyze` subcommand can be run multiple times with different configurations if
-the user would like to sort and filter the results using different objectives or
-under different constraints.
-
-### Examples
```

Comment:
Are there examples still worth keeping, but showing it in the context of the profile step?

Reply:
They already exist for the profile step and are redundant (see line 166)